app/build.gradle

```diff
@@ -1,16 +1,23 @@
 apply plugin: 'com.android.application'
+apply from: '../config/quality.gradle'
+
+
+static def releaseTime() {
+    return new Date().format("MMM dd yyyy", TimeZone.getTimeZone("UTC"))
```

Comment:
Please don't add the date and time to the app filename, let's work with semantic versioning.